docs/internal-documentation/maintenance/ReleaseProcess-deb-packages.md

```diff
@@ -0,0 +1,106 @@
+Release Process - Create .deb Packages
+=====================================
+
+
+## Overview
+
+The process to build the packages is split in two parts. First we update the
```

Comment:
> I think it would be better to use a passive description, not "we", The "nightly packages" are updated

fixed

> Is "updated" correct? Not "built"?

When I say updated I am referring to the source of packages (all the files in de `debian` folders)

> I also find this "overview" section very detailed. This is more "Package building". Could there be an overview that the overall process? What is the goal of the process?

I have updated the overview